ironfish/src/network/messages/getBlocks.test.ts

```diff
@@ -15,112 +16,77 @@ describe('GetBlocksRequest', () => {
 })
 
 describe('GetBlocksResponse', () => {
-  it('serializes the object into a buffer and deserializes to the original object', () => {
+  const nodeTest = createNodeTest()
+
+  function expectGetBlocksResponseToMatch(a: GetBlocksResponse, b: GetBlocksResponse): void {
```

Comment:
[This method](https://github.com/iron-fish/ironfish/blob/67da183ca994987db05e3b061dec6768f8859f3c/ironfish/src/primitives/block.ts#L70) seems pretty comprehensive. Wondering if we can reuse logic for these expect functions

Reply:
Changed to reuse `Block.equals()`